src/ucm/cuda/cudamem.c

```diff
@@ -211,18 +214,31 @@ static ucm_mmap_hook_mode_t ucm_cuda_hook_mode()
     return ucm_get_hook_mode(ucm_global_opts.cuda_hook_mode);
 }
 
+static const char* ucm_cuda_hook_mode_str()
+{
+    switch (ucm_cuda_hook_mode()) {
+        case UCM_MMAP_HOOK_BISTRO :
+            return UCM_MMAP_HOOK_BISTRO_STR;
+        case UCM_MMAP_HOOK_RELOC:
+            return UCM_MMAP_HOOK_RELOC_STR;
+        case UCM_MMAP_HOOK_NONE:
+            return "none";
+        default:
+            return "unknown";
+    }
+}
+
 static ucs_status_t
-ucm_cuda_install_hooks(ucm_cuda_func_t *funcs, int *used_reloc,
-                       const char *name)
+ucm_cuda_install_hooks(ucm_cuda_func_t *funcs, const char *name)
 {
-    const char UCS_V_UNUSED *hook_mode;
-    unsigned num_bistro, num_reloc;
+    const ucm_mmap_hook_mode_t hook_mode = ucm_cuda_hook_mode();
+    const char *hook_mode_str = ucm_cuda_hook_mode_str();
+    unsigned num_hooked;
     ucm_cuda_func_t *func;
     ucs_status_t status;
     void *func_ptr;
 
```

Comment:
`if (hook_mode == UCM_MMAP_HOOK_NONE) { return }`

Reply:
`ucm_cudamem_install()` already does that check. Do we need it again here?

Reply:
missed it. then we are good.